package.json

```diff
@@ -15,7 +15,7 @@
   "author": "Felipe Sabino <felipe@sabino.me>",
   "license": "MIT",
   "dependencies": {
-    "kafka-node": "0.2.27",
+    "kafka-node": "0.3.2",
```

Comment:
👍 

Oh, did you noticed that 

- it seems that Keyed Partitioner is know [embedded in the client](https://github.com/SOHU-Co/kafka-node/pull/260/files#diff-45c97e793d18dbc47540eadfdb25b6c9R47) after 0.3.0?
- After 0.3.1 the added [Buffer batch](https://github.com/SOHU-Co/kafka-node/pull/262) which (maybe) replaces the usage of async.cargo on producer clients as well?

😍